src/Types/ContractName.cs

```diff
@@ -36,6 +36,12 @@ public static bool TryParse(string name, out (ContractName? ContractName, Valida
         return validate;
     }
 
+    /// <summary>
+    /// Get the contract name part of <see cref="Name"/>.
+    /// </summary>
+    /// <returns>Contract name</returns>
+    public string GetContractName() => this.Name[(this.Name.IndexOf('_') + 1)..];
```

Comment:
This should probably be named differently, since it won't behave like the `GetContractName` method on `ReceveName` (after addressing my comment there).